packages/js/components/src/table/style.scss

```diff
@@ -86,6 +86,10 @@
 		justify-content: flex-end;
 		display: flex;
 	}
+
+	&__table-preface {
+		margin: $gap-small $gap $gap-small $gap;
+	}
```

Comment:
Personal preference, so not a blocker, but sharing just in case. :smile: I usually prefer having this kind of selector separately as a completely independent selector:

```CSS
.woocommerce-table__table-preface {
	margin: $gap-small $gap $gap-small $gap;
}
```

This way, it's possible to search the codebase by `woocommerce-table__table-preface` and find the relevant styles.

---

I also wonder if the class name should be `woocommerce-table__preface` instead of repeating `table`? :thinking:

Reply:
> I also wonder if the class name should be woocommerce-table__preface instead of repeating table?

Good point. I think "table" word is redundant now that you mentioned it, so I removed it here [96496f2](https://github.com/woocommerce/woocommerce/pull/53495/commits/96496f2a864ff90b540d47d1ce62dbe0ba60900d). I also moved the selector out to be a standalone class instead of nesting it under `.woocommerce-table. Thanks again for your suggestions!